lib/node_modules/@stdlib/string/base/remove-first-code-point/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@stdlib/string/base/first-code-point",
+  "version": "0.0.0",
+  "description": "Removes the first `n` Unicode code points of a string.",
```

Comment:
```suggestion
  "description": "Remove the first Unicode code points of a string.",
```

Reply:
These packages are a little special, thus far, in how the `package.json` descriptions are handled, where we try to avoid including code blocks.